app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -272,7 +272,15 @@
 //
 //  Field
 //  ---------------------------------------------
-
+.login-content {
+    .admin__field {
+         label[for="captcha"] > span {
```

Comment:
Styles for captcha functionality should be in the Captcha module instead of Backend module

Reply:
I have removed captcha style from the _fields.less